JavaScript/d-obj-create.js

```diff
@@ -156,11 +156,16 @@ creaters.closureCreate = function closureCreate() {
 // Testing
 if (module.parent === null) {
   const benchmark = require('./2-benchmark.js');
-  console.log('Creation + using arrays. Size = ' + creaters.arraySize + ':\n');
+  console.log('Creating arrays. Size = ' + creaters.arraySize + ':\n');
   benchmark.do(1000, 3, [
     creaters.mixinCreate,
     creaters.classCreate,
     creaters.prototypeCreate,
     creaters.closureCreate
+  ], [
+    'mixinCreate',
+    'classCreate',
+    'prototypeCreate',
+    'closureCreate'
   ]);
-}
+}
```

Comment:
I think you should configure your editor properly so that it

a) inserts newlines at end of files automatically and doesn't even show them to you, like any source code editor is actually supposed to do, and
b) runs eslint automatically on each save and shows you warnings and errors on the left of lines that have some issues, like any source code editor can do if you tell it to :)